installer/templates/phx_ecto/data_case.ex

```diff
@@ -0,0 +1,65 @@
+defmodule <%= application_module %>.DataCase do
```

Comment:
Should this be a `DataCase` or rather `DatabaseCase` - it's closely related to ecto, so I don't think we should try making it abstract to general concept of data. There might be many sources of data in an application and not all should use this case template.